package.json

```diff
@@ -82,6 +82,7 @@
     "eslint-plugin-spellcheck": "0.0.11",
     "exceljs": "3.3.1",
     "fibers": "^5.0.0",
+    "font-carrier": "^0.2.0",
```

Comment:
Do we need this dependency if we install it in task? https://github.com/DevExpress/DevExtreme/pull/14992/files#diff-5250c516fa2c2d5847d45893291199339489509cd386afd23d52089d84c64b16R203